docs/documenting-schema.rst

```diff
@@ -0,0 +1,73 @@
+.. _documenting-schema:
+
+Documenting schema
```

Comment:
```suggestion
Documenting the schema
```

Or 

```suggestion
Documenting the GraphQL schema
```

Reply:
I'm okay with either of these, but I changed this line in response to direct feedback from @rafalp.  So I'd like to get input from him before selecting one of these.

I also still like "Documenting your schema", myself.  🙂

Reply:
`Documenting the schema` is okay.